src/xdocs/config_filters.xml

```diff
@@ -860,6 +855,60 @@ public class UserService {
 &lt;suppress-xpath checks=&quot;.*&quot; query=&quot;/CLASS_DEF[@text='TestClass']
           //METHOD_DEF[@text='testMethod']//VARIABLE_DEF[@text='testVariable']&quot;/&gt;
         </source>
+        <p>
+          In the following sample, violations for <code>LeftCurly</code> check will be suppressed
+          for classes with name <i>Main</i> or for methods with name <i>calculate</i>.
+        </p>
+        <source>
+&lt;suppress-xpath checks=&quot;LeftCurly&quot; query=&quot;/CLASS_DEF[@text='Main']/OBJBLOCK/LCURLY
+          | //METHOD_DEF[@text='calculate']/SLIST&quot;/&gt;
```

Comment:
users will be lazy to make xpath very specific , can we make xpath like:
`/CLASS_DEF[@text='Main']/* | //METHOD_DEF[@text='calculate']/*` 
to  not bother user to know what token Check actually report an violation.

Reply:
done